lib/mutant/reporter/cli/tput.rb

```diff
@@ -7,13 +7,15 @@ class Tput
 
         private_class_method :new
 
-        # Return detected tput support
+        # The detected tput support
```

Comment:
I wonder if many cases where you used `The` as a prefix on the method description could simply be removed without affecting comprehensibility?

A method named `self.detect` with a description `Detected tput support` seems just as reasonable as this.